src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/Junit5FunctionalSpec.groovy

```diff
@@ -85,4 +86,18 @@ class Junit5FunctionalSpec extends AbstractPitestFunctionalSpec {
             result2.standardOutput.contains('Reusing configuration cache.')
     }
 
+    @Issue("https://github.com/szpak/gradle-pitest-plugin/issues/333")
+    @Ignore("This requires Gradle 8.1, and its Groovy is too modern for Spock 2.3-groovy-2.5, and Gradle 6.9.2 is too old for 2.3-groovy-3.0.")
```

Comment:
The problem is that you enforce Gradle 8.x which uses Groovy 3 and nebula plugin by default passes "all" project classpath, including test dependencies, so also Spock 2.3-groovy-2.5 (because the plugin is compiled with Gradle 6.x) which doesn't like Groovy 3. As that test project deals only with JUnit 5, Groovy is definitely not needed and can be filtered out (as I already do in `PitestPluginGradleVersionFunctionalSpec`).

See:
> Caused by: org.gradle.groovy.scripts.ScriptCompilationException: Could not compile initialization script '/home/foobar/Code/gradle/gradle-pitest-plugin/build/nebulatest/info.solidsoft.gradle.pitest.functional.Junit5FunctionalSpec/should-not-reference-project-data-at-execution-time/.gradle-test-kit/init.gradle'

Btw, I'm not sure where to put that test, but - for now - let's keep it here.